### PR TITLE
Rickithadi/tests

### PR DESCRIPTION
**Pull request by rickithadi**
**CHANGELOG** :memo:

*

**PRINTS(OPTIONAL)** :framed_picture:

**LABELS**

* Example: enhancement, wip etc

### CODE REVIEW

**Comment by rickithadi** [comment]
https://github.com/pyymenta/spacecraft-cli/issues/5

**Comment by pyymenta** [comment]
@rickithadi thanks about your contribution, I'll review it!

File: tests/spacecraft.spec.ts [hunk 1]
```diff
@@ -1,9 +1,22 @@
...
   });
+  it('moves forward', () => {
+    expect(spaceCraft.forward(spaceCraft.y)).toEqual(1);
+  });
+  it('moves left', () => {
+    expect(spaceCraft.left(spaceCraft.x)).toEqual(-1);
+  });
+  it('moves right', () => {
+    expect(spaceCraft.right(spaceCraft.x)).toEqual(1);
+  });
```
Comment (pyymenta) [inline tests/spacecraft.spec.ts:17]: Typo here `bacl`

Could you fix it?

Comment (rickithadi) [reply tests/spacecraft.spec.ts:17]: Sure thing

**Review by pyymenta — APPROVED** [review]
LGTM!! 🚀🚀🚀

**Pull request merged**